### PR TITLE
Remove Ucommerce.Runtime completely and move the relevant bits to new Api

### PR DESCRIPTION
**Pull request by skovsboll**
This PR goes with https://github.com/Ucommercenet/Ucommerce/pull/251

I would like some feedback on the contents, as I'm in unknown territory here.

### CODE REVIEW

File: src/AvenueClothing/Controllers/EmailController.cs [hunk 1]
```diff
@@ -20,30 +18,30 @@ public class EmailController : RenderMvcController
...
+                    VariantSku = orderLine.VariantSku,
+                    Total = new Money(orderLine.Total.GetValueOrDefault(),
+                        orderLine.PurchaseOrder.BillingCurrency.ISOCode).ToString(),
+                    Tax = new Money(orderLine.VAT, basket.BillingCurrency.ISOCode).ToString(),
+                    Price = new Money(orderLine.Price, basket.BillingCurrency.ISOCode).ToString(),
+                    PriceWithDiscount = new Money(orderLine.Price - orderLine.Discount, basket.BillingCurrency.ISOCode)
+                        .ToString(),
+                    Quantity = orderLine.Quantity,
+                    Discount = orderLine.Discount
+                };
```
Comment (ilny) [inline src/AvenueClothing/Controllers/EmailController.cs:44]: Funny looking diff, it's however correct.

File: tools/Deploy/DemoStoreDeploy.cmd [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 cd ..\..\
 
-SET version=6.0.3.20724
+SET version=7.0.2.20724
```
Comment (ilny) [inline tools/Deploy/DemoStoreDeploy.cmd:5]: Unnecessary change, please revert or go with 7.0.0

File: src/AvenueClothing/Controllers/ProductController.cs [hunk 3]
```diff
@@ -127,37 +124,5 @@ into g
...
-            //     variantSku = variant.VariantSku;
-            // }
-            //
-            // // Only use the current product where there are no variants
-            // else if (!product.Variants.Any())
-            // {
-            //     variantSku = product.Sku;
-            // }
-
-            return variantSku;
```
Comment (ilny) [inline src/AvenueClothing/Controllers/ProductController.cs:160]: Where did this section go? Is the old functionality preserved elsewhere?

Comment (skovsboll) [reply src/AvenueClothing/Controllers/ProductController.cs:160]: It's actually only used here these days: https://github.com/Ucommercenet/Avenue-Clothing-For-Umbraco/blob/c8c95aed15757ac097c8d2fe3dbdb033c51864ff/src/AvenueClothing/Api/AvenueClothingApiProductController.cs#L60
I think this is old remains of a former civilization.

File: tools/Deploy/DemoStoreDeploy.cmd (hunk 2 above)
Comment (skovsboll) [reply tools/Deploy/DemoStoreDeploy.cmd:5]: Ok!

File: src/AvenueClothing/Controllers/ProductController.cs (hunk 3 above)
Comment (skovsboll) [reply src/AvenueClothing/Controllers/ProductController.cs:160]: Used here only:

https://github.com/Ucommercenet/Avenue-Clothing-For-Umbraco/blob/c8c95aed15757ac097c8d2fe3dbdb033c51864ff/src/AvenueClothing/Api/AvenueClothingApiProductController.cs#L60

Comment (ilny) [reply src/AvenueClothing/Controllers/ProductController.cs:160]: Fair enough.
